### ISSUE TITLE
IE10: Unexpected call to method or property access

### ISSUE THREAD

**Issue by jetlogs** [videojs/video.js#4378]
## Description
JS error on IE10 with HTML5 tech causing entire JS execution to break

## Steps to reproduce
1. ~~multiple videoJS instances on same page~~ reproducible even with a single player instance
2. when one video instance ends, JS error gets triggered

## Results
```
SCRIPT65535: Unexpected call to method or property access.
audio-track-list.js (44,1)
```
![](http://i.imgur.com/tS93zlA.jpg)

### Expected
```js
var disableOthers = function disableOthers(list, track) {
  for (var i = 0; i < list.length; i++) {
    if (track.id === list[i].id) {
      continue;
    }
    // another audio track is enabled, disable it
    list[i].enabled = false;
  }
};
```
in this function, list[i] would return an empty Object {} without an id property causing said issue
causes the rest of the JS execution to break when this triggers

any way to disable the disableOthers() function or fix this issue? I'm not using any audioTracks for the videos.

Thanks!

## Additional Information
IE10
videojs: 5.16.2

**Comment by jetlogs** [comment]
update:
it looks like this happens even with only one instance of the player on IE10

**Comment by jetlogs** [comment]
created a PR to fix this issue on v6.2
https://github.com/videojs/video.js/pull/4395
